examples/plugins/README.rst

```diff
@@ -14,3 +14,9 @@ And to remove them on a development environment, you may run, at the
 same directory::
 
  $ python setup.py develop --uninstall --user
+
+Second way to add to install plugin is to use Makefile's link target,
+Simply add symlink of target plugin to parent avocado directory and
+do make link::
+ $ ln -s examples/plugins/job-pre-post/mail ../mail-plugin
+ $ make link
```

Comment:
Oh. Crap. This update should not be bound to that pull request. This is just my temproral crap.
I thought that github-pull requests bound speciffic commit ID on pull creation